### PR TITLE
Fixes https://issues.redhat.com/browse/WFLY-19460

### PR DESCRIPTION
**Pull request by HarshaVardhan-Sai**
Updated the CONTRIBUTING.MD file for more readability to fix https://issues.redhat.com/browse/WFLY-19460

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,4 +12,20 @@ If you like our project, but just don’t have time to contribute, that’s fine
...
+
+To create a new issue, comment on an existing issue, or assign an issue to yourself, you'll need to first [create a JIRA account](https://issues.redhat.com/).
+
+# Good First Issues
+
+---
+
+Check out our issues with the `good-first-issue` label. These are a triaged set of issues that are great for getting started on our project. These can be found [here](https://issues.redhat.com/issues/?filter=12403174).
+
+Once you have selected an issue you'd like to work on, make sure it's not already assigned to someone else. To assign an issue to yourself, simply click on "Start Progress". This will automatically assign the issue to you.
```
Comment (bstansberry) [inline CONTRIBUTING.md:29]: Can you add this sentence:

If you're not able to assign it to yourself that way, post a message in [the wildfly-developers Zulip stream](https://wildfly.zulipchat.com/#narrow/stream/174184-wildfly-developers) and someone will help get it assigned to you.

**Review by bstansberry — CHANGES_REQUESTED** [review]
Thanks @HarshaVardhan-Sai -- I just have one small change I'd like, since the issue assignment stuff has issues sometimes.

File: CONTRIBUTING.md (hunk 1 above)
Comment (HarshaVardhan-Sai) [reply CONTRIBUTING.md]: @bstansberry Implemented it.

**Review by bstansberry — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @HarshaVardhan-Sai

**Pull request merged**